makefile

```diff
@@ -57,3 +57,16 @@ pull_chunked : submodules/chunked_requests
 	@echo ""
 	@echo "Pulling down updates from chunked_requests"
 	cd submodules/chunked_requests; git pull origin master
+
+update_offline :
```

Comment:
can we name this to be a tiny bit more informative `update_plotlyjs_for_offline` ?